vendor/priors/gwt.js

```diff
@@ -646,8 +646,27 @@ function createVideoThumbnailProm(future, asyncReader, fileSize, fileName) {
             console.log(e);
             future.complete("");
         }
-        let blob = new Blob([new Uint8Array(bytes)], {type: "octet/stream"});
-        let url = window.URL.createObjectURL(blob);
+        //        let blob = new Blob([new Uint8Array(bytes)], {type: "octet/stream"});
+        //safari is picky about mime type
+        let safari =
+          /constructor/i.test(window.HTMLElement) ||
+          !!window.safari ||
+          !!window.WebKitPoint
+        var type = "octet/stream";
+        let fileNameLowercase = fileName.toLowerCase();
+        if (safari) {
+            if (fileNameLowercase.endsWith('.mov') ) {
+                type = "video/quicktime";
```

Comment:
Can't we use the supplied mimetype generated from the actual bytes?

Reply:
fixed